pkg/florist/user_test.go

```diff
@@ -30,3 +32,20 @@ func TestUserSystemAddSuccessVM(t *testing.T) {
 
 	assert.NoError(t, err, "florist.UserSystemAdd")
 }
+
+func TestUserSystemAddWithUIDSuccessVM(t *testing.T) {
+	florist.SkipIfNotDisposableHost(t)
+
+	userName := "maniglia"
```

Comment:
minor: I can't use the same username and home directory as in previous test. The `maniglia` user is already created on the VM and has different UID.. All tests are passing if I change the username to: `foobar` and adjust the home dir accordingly (`/opt/foobar`)